src/cli/repo/s3.go

```diff
@@ -183,7 +183,7 @@ func connectS3Cmd(cmd *cobra.Command, args []string) error {
 		return Only(ctx, errors.Wrap(err, "Failed to parse m365 account config"))
 	}
 
-	r, err := repository.Connect(ctx, a, s, options.Control())
+	r, err := repository.Connect(ctx, a, s, true, options.Control())
```

Comment:
@ryanfkeepers @neha-Gupta1 - thoughts on just adding a `func ConnectWrapper(...)` helper in the `repo` package that connects to the repo and fires the event? Seems cleaner.

Reply:
Seems to be a cleaner way. If @ryanfkeepers agrees with it.

Reply:
@neha-Gupta1 - to save you the wait till tomorrow - how about we move forward with that approach. I think @ryanfkeepers will be on board and if not I will owe you both some ☕

Reply:
I am in such deep coffee debt.